common/tools/eslint-plugin-azure-sdk/ci.yml

```diff
@@ -0,0 +1,79 @@
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+      
+trigger:
+  branches:
+    include:
+      - master
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/eslint-plugin-azure-sdk
+
+pr:
+  branches:
+    include:
+      - master
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/eslint-plugin-azure-sdk
+
+variables:
+  NodeVersion: "10.x"
+  skipComponentGovernanceDetection: true
+
+stages:
+  - stage: Build
+    jobs:
+      - job: Build
+
+        pool:
+          vmImage: ubuntu-16.04
+
+        steps:
+          - task: NodeTool@0
+            inputs:
+              versionSpec: "$(NodeVersion)"
+            displayName: "Install Node.js $(NodeVersion)"
```

Comment:
does this map to the pattern of CI that we use for other internal packages?

Reply:
I forgot to remove this file. I was going to clone the one for test-recorder and make it fit in. I'm pretty sure this is why the ci below failed.